server/src/main/java/com/senlainc/warsaw/tyurin/entity/Order.java

```diff
@@ -2,20 +2,32 @@
 
 import com.senlainc.warsaw.tyurin.util.OrderStatus;
 
+import javax.persistence.*;
 import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 
+@Entity
+@Table(name = "order")
 public class Order {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
+    @Column(name = "price")
     private double price;
+    @Column(name = "submission_date")
     private LocalDateTime submissionDate;
+    @Column(name = "start_date")
     private LocalDateTime startDate;
+    @Column(name = "completion_date")
     private LocalDateTime completionDate;
+    @Transient
     private OrderStatus orderStatus;
+    @Transient
     private List<Long> craftsmenId;
```

Comment:
Now when you working with hibernate let's add relationships between Entities (One to Many, Many to Many, etc.) and you can change to entity lists

Reply:
use OneToMany annotation and reverse one on second entity to create relationships between them